eXo/Sources/Controllers/Setting/ServerEditViewController.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2003-2015 eXo Platform SAS.
+//
+// This is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of
+// the License, or (at your option) any later version.
+//
+// This software is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this software; if not, write to the Free
+// Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+// 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+
+import UIKit
+
+class ServerEditViewController: UIViewController {
+
+    
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var deleteButtonConstraintToBottom: NSLayoutConstraint!
+    let kDeleteButtonBottomMargin:CGFloat = 20.0
+    
+    @IBOutlet weak var textView: PlaceholderTextView!
+    var server:Server!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.textView.text = server.serverURL
+        textView.placeholder = NSLocalizedString("OnBoarding.Message.EnterURL", comment: "")
+        Tool.applyBorderForView(self.deleteButton)
+
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector("keyboardWillShow:"), name:UIKeyboardWillShowNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector("keyboardWillHide:"), name:UIKeyboardDidHideNotification, object: nil)
+
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    @IBAction func saveAction(sender: AnyObject) {
+        save()
+    }
+    
+    @IBAction func deleteAction(sender: AnyObject) {
+        let alertController = UIAlertController(title: NSLocalizedString("Setting.Title.DeleteServer", comment: ""), message: NSLocalizedString("Setting.Message.DeleteServer", comment: ""), preferredStyle: UIAlertControllerStyle.Alert)
```

Comment:
Same as with `saveAction() => save()`, you can create a method `delete()` and call it from here.